docs/tutorials/material_solvated.ipynb

```diff
@@ -117,7 +117,7 @@
    "metadata": {},
    "source": [
     "However, now we will construct a component object (for type `MaterialSolvated`), based on the knowledge that there is a 75 % surface coverage of the silicon block by the polymer film. \n",
-    "Note that this object takes the two materials and then the fractional amount of the second material in the first."
+    "Note that this object takes the material (polymer film) and a solvent, which both are instances of `Material`. The ratio between these are given by the solvent fraction (the remaining 25 % surface coverage)."
```

Comment:
I wouldn't use the word ratio here, as I think it makes it slightly confusing. Keeping everything as a fraction or percentage would be clearer.

"Note that this object takes the material (polymer film) and a solvent, which both are instances of `Material`. The fraction of material and solvent is given by the the `solvent_fraction`, which in this case is 25% solvent."

Reply:
changed